salt/metalk8s/salt/master/installed.sls

```diff
@@ -7,9 +7,9 @@
 
 {%- set ip_candidates = salt.network.ip_addrs(cidr=networks.control_plane) %}
```

Comment:
Same as above, should be using the grain.

Reply:
oups